javaslang/src/main/java/javaslang/collection/BitMappedTrie.java

```diff
@@ -83,22 +81,19 @@ private static int treeSize(int branchCount, int depthShift) {
         }
     }
 
-    private BitMappedTrie<T> boxed() { return map(identity()); }
+    BitMappedTrie<T> boxed() { return map(identity()); }
```

Comment:
O(n) for changing the type (which should be static in a statically typed language) IS EVIL.